advanced/Scripts/webpage.sh

```diff
@@ -211,6 +211,10 @@ SetExcludeClients(){
 
 }
 
+Halt(){
+        nohup bash -c "sleep 5; halt" &> /dev/null </dev/null &
```

Comment:
Now I have found the time to think about this: In general, it seems to be best practice to use the `shutdown` command instead of `halt` on Linux and UNIX as it e.g. sends around messages to all users that are logged in to notify them of a shutdown or reboot so they can properly finish what they are working on or save their work.

You have to take into account that this will be implemented into Pi-hole and not all users run it on a Pi-like device. I'll have to change the `reboot` command as well, now that I think about it.

Also, you won't need `sleep 5` anymore, because it has an internal timer and will also prevent users from loging in any more (since the system will shut down soon, anyway).